src/components/contextual/pages/vebal/LMVoting/utils.ts

```diff
@@ -0,0 +1,10 @@
+import { orderedPoolTokens } from '@/composables/usePool';
+import { VotingGaugeWithVotes } from '@/services/balancer/gauges/gauge-controller.decorator';
+import { Pool } from '@/services/pool/types';
+
+export function orderedTokenURIs(gauge: VotingGaugeWithVotes): string[] {
+  const sortedTokens = orderedPoolTokens(gauge.pool as Pool, gauge.pool.tokens);
+  return sortedTokens.map(
+    token => gauge.tokenLogoURIs[token?.address || ''] || ''
+  );
+}
```

Comment:
Not a massive fan of nesting helpers like this in the components folder. Seems like this would make more sense as part of a useGauge composable or helpers file elsewhere.